pkg/executor/aggregate/agg_hash_executor.go

```diff
@@ -304,12 +304,13 @@ func (e *HashAggExec) initPartialWorkers(partialConcurrency int, finalConcurrenc
 			globalOutputCh:       e.finalOutputCh,
 			partialResultsMap:    partialResultsMap,
 			groupByItems:         e.GroupByItems,
-			chk:                  exec.TryNewCacheChunk(e.Children(0)),
-			groupKeyBuf:          *groupKeyBuf,
-			serializeHelpers:     aggfuncs.NewSerializeHelper(),
-			isSpillPrepared:      false,
-			spillHelper:          e.spillHelper,
-			inflightChunkSync:    e.inflightChunkSync,
+			// chk:                  e.NewChunkWithCapacity(e.RetFieldTypes(), 0, e.MaxChunkSize()),
```

Comment:
Can you remove it ?

Reply:
> Can you remove it ?

The ci always fails, I'm trying to figure out which leads to the fail.